### PR TITLE
Complete the big merge

### PR DESCRIPTION
**Pull request by ubermichael**
Fixes up the display of periods to include items from the period, merges old generated template code with new template code, and fixes the tests.

### CODE REVIEW

File: templates/language/search.html.twig [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% extends 'base.html.twig' %}
 
-{% block title %}Language Search{% endblock %}
+{% block title %}Category Search{% endblock %}
```
Comment (joeytakeda) [inline templates/language/search.html.twig]: I think this is a typo?

File: templates/subject/search.html.twig [hunk 2]
```diff
@@ -13,13 +12,77 @@
  {% endblock %}
 
 {% block body %}
+
+    <form method="get" action="{{ path('subject_search') }}" class="form-horizontal">
```
Comment (joeytakeda) [inline templates/subject/search.html.twig]: Any reason not to use `partial/search-form.html.twig`?

**Review by joeytakeda — COMMENTED** [review]
One typo and one place where we could use a partial.

File: templates/language/search.html.twig (hunk 1 above)
Comment (ubermichael) [reply templates/language/search.html.twig]: I love a good copy/pasta.

File: templates/subject/search.html.twig (hunk 2 above)
Comment (ubermichael) [reply templates/subject/search.html.twig]: I copied it from material/search.html.twig.

**Pull request merged**